docker-compose.yml

```diff
@@ -133,4 +140,7 @@ services:
       dockerfile: src/Web/WebStatus/Dockerfile
 
   dapr-placement:
-    image: "daprio/dapr:1.9.4"
\ No newline at end of file
+    image: "daprio/dapr:1.12.3"
+    command: ["./placement", "--port", "50006"]

```

Comment:
```suggestion
    command: ["/placement", "--port", "50006", "--inmem-store-enabled", "true", "--initial-cluster", "dapr-placement-0=127.0.0.1:50006", "--id", "dapr-placement-0"]
```

Reply:
Do not need manual command anymore, just simplifying choosing defaults